x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.test.tsx

```diff
@@ -247,37 +249,5 @@ describe('APIKeysGridPage', () => {
       expect(await findByText('You do not have permission to create API keys')).toBeInTheDocument();
       expect(queryByText('Create API key')).toBeNull();
     });
-
```

Comment:
Since we are mocking the data from ES it will show whatever we put in the mock. Looking at the `routes/api-keys/get.ts` it passes an `isAdmin` value which I believe is the flag that returns all just the current users.